eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/model/Comment.java

```diff
@@ -0,0 +1,37 @@
+package hu.flowacademy.companycalendar.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Data
+@Entity
+@Table
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Comment{
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column
+    private String content;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne
+    @JsonIgnore
+    private Meeting meeting;
+
+
```

Comment:
Formazzuk kicsit, nem kell ennyi space ide. Mi a standard it @borosr?

Reply:
A 4 space hosszú tab a gond vagy az az űr, ami az utolsó attributum után van?